src/qemu/qemu_capabilities.c: Fix qemu-system-* command line

Using -no-graphic -daemonize on Qemu command line doesn't do exactly what it
seems to do (for instance, it redirects some outputs to stdio) and it has even
been disabled on newer Qemu (qemu git commit ab51b1d568e02c80b1abf9016bda3a86dc1db389)
As a solution, just make sure that everything is redirected to null.

Signed-off-by: Arnaud Patard <apatard@hupstream.com>

Index: libvirt-1.0.2/src/qemu/qemu_capabilities.c
===================================================================
--- libvirt-1.0.2.orig/src/qemu/qemu_capabilities.c
+++ libvirt-1.0.2/src/qemu/qemu_capabilities.c
@@ -2358,6 +2358,11 @@ qemuCapsInitQMP(qemuCapsPtr caps,
                                "-nographic",
                                "-M", "none",
                                "-qmp", monarg,
+			       "-serial", "null",
+			       "-parallel", "null",
+			       "-monitor", "none",
+			       "-display", "none",
+			       "-vga", "none",
                                "-pidfile", pidfile,
                                "-daemonize",
                                NULL);
